lib/Xliff.js

```diff
@@ -816,6 +816,11 @@ Xliff.prototype.deserialize = function(xml) {
                             tu.resname = tu.source["$t"];
                         }
 
+                        if (tu.source["x-key"]) {
+                             tu.source["$t"]= tu.source["x-key"];
+                             tu.resname = tu.source["x-key"];
+                        }
+
```

Comment:
Also, all of the LG xliffs are in 2.0 format, right? Do we really need to add this to the 1.2 xliff reader?

Maybe this x-key should be in the 2.0 branch. (BTW - that branch is pushed so you can look at it, but it is not finished yet.)

Reply:
At this point, Yes. Regarding  Xliff 2.0 support it's not ready to be used. So In order to check and verify loctool in webOS,  I need this to add in xliff reader the current version.

Reply:
I'll check your 2.0 branch. As I shared through email,`x-key` attribute is removed in xliff 2.0 then matches the `name` attribute in `unit` tag. such as
```
<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0" srcLang="en-KR" trgLang="ko-KR">
 <file id="settings_f1" original="settings">
  <group id="settings_g1" name="javascript">
   <unit id="settings_140" name="pictureModeSettings">
    <segment>
     <source>Customize</source>
     <target>상세 설정</target>
    </segment>
   </unit>
...
```